pavlolucik/docker/05buildSameWithMaven/gw-spring-boot/.bomr/bomr.yaml

```diff
@@ -0,0 +1,32 @@
+bomr:
```

Comment:
We are really need these files in a hidden folder in this repo and all other source files???